### PR TITLE
added back_left and back_right

### PR DESCRIPTION
**Pull request by HannesBachter**
needed for https://github.com/ipa320/ipa_navigation_planning/pull/128

### CODE REVIEW

File: cob_default_robot_config/cob4-2/torso_joint_configurations.yaml [hunk 1]
```diff
@@ -11,7 +11,9 @@ front_up_full: [[3.14, -1.57]]
 front_down: [[-1.0, 0.5]]
 front_down_full: [[-3.14, 1.57]]
 
-back: [[0.0, 3.14]]
+back: [back_left]
```
Comment (benmaidel) [inline cob_default_robot_config/cob4-2/torso_joint_configurations.yaml:14]: is this really working?

**Review by benmaidel — CHANGES_REQUESTED** [review]

File: cob_default_robot_config/cob4-2/torso_joint_configurations.yaml (hunk 1 above)
Comment (HannesBachter) [reply cob_default_robot_config/cob4-2/torso_joint_configurations.yaml:14]: have not tried that - will do

Comment (fmw-hb) [inline cob_default_robot_config/cob4-2/torso_joint_configurations.yaml]: does work in simulation...

**Comment by HannesBachter** [comment]
works in simulation

File: cob_default_robot_config/cob4-2/torso_joint_configurations.yaml (hunk 1 above)
Comment (floweisshardt) [reply cob_default_robot_config/cob4-2/torso_joint_configurations.yaml:14]: yes, this is a valid trajectory specification, see `nod` and `shake`

**Review by floweisshardt — APPROVED** [review]
CR: OK

**Review by benmaidel — APPROVED** [review]

**Pull request merged**